spec/bowlingcardSpec.js

```diff
@@ -0,0 +1,132 @@
+describe('Bowlingcard', function() {
+  var bowlingcard;
+  beforeEach(function() {
+    bowlingcard = new Bowlingcard();
+  });
+
+  describe('.score', function() {
+    it('starts game at 0', function() {
+      expect(bowlingcard.score).toBe(0);
+    });
+  });
+
+  describe('.enterScore', function() {
+    it('records the users score', function() {
+      bowlingcard.enterScore(5);
```

Comment:
Naming: Is the player entering a score or a roll? Naming sounds trivial, but it's worth thinking about it at the start, to avoid confusions later.